src/__tests__/BezierCurve.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { BezierCurve } from "canvas/Geometry/BezierCurve";
+import { Point } from "canvas/Geometry/Point";
+
+test("splitting bezier curve", async () => {
+    const curve = new BezierCurve(new Point(0, 0), new Point(2, 0), new Point(1, 2));
+
+    const parts = curve.split(new Point(1, 1));
+    expect(parts[0].getStart()).toEqual(curve.getStart());
+    expect(parts[0].getEnd()).toEqual(parts[1].getStart());
+    expect(parts[1].getEnd()).toEqual(curve.getEnd());
+
+});
```

Comment:
Maybe we can change the name of the file __tests__ to _tests

Reply:
Looking around I saw three ways of organizing tests: the way I used with a __tests__ folder inside src, then I also saw people to put the tests in the same folder as the class they test (which I thought would crowd our source code) and finally some put the tests in a folder named "test" at the same level as src (but I felt like the tests related strongly to the source code and so would belong to the folder)  

I used the name __tests__ because that's what I saw people use when they have a folder inside src. But I tried and it looks like Jest will find the tests regardless of the name of the folder. So I don't have any strong opinions on the name of the folder, I can rename it to _tests.

Reply:
I am good with the organization you have. I think the the name _tests is easier to type because it has fewer underscores. It also just looks better to me.